### ISSUE TITLE
weresync fails with "Error: /dev/sdb: unrecognised disk label"

### ISSUE THREAD

**Issue by nocnokneo** [DonyorM/weresync#7]
WereSync Version [0.2]: 0.2.1
Name of Distribution (ex. Ubuntu): CentOS
Version of Distribution (ex. 16.04): 7
Python version (ex. 3.4): 3.4

Expected Behavior:
This works:

`sudo weresync --verbose -C --boot-partition 2 --efi-partition 1 /dev/sda /dev/sdb`

`/dev/sda` is the source 1 TB SSD
`/dev/sdb` is a brand new 500 GB SSD, just out of the box.

Actual Behavior:

```
$ sudo pip3 install weresync
Collecting weresync
  Downloading WereSync-0.2.1-py3-none-any.whl (70kB)
    100% |████████████████████████████████| 71kB 1.4MB/s 
Collecting parse>=1.6.6 (from weresync)
  Using cached parse-1.8.0.tar.gz
Installing collected packages: parse, weresync
  Running setup.py install for parse ... done
Successfully installed parse-1.8.0 weresync-0.2.1
You are using pip version 8.1.2, however version 9.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
$ sudo weresync --verbose -C --boot-partition 2 --efi-partition 1 /dev/sda /dev/sdb
Checking partition validity.
Traceback (most recent call last):
  File "/bin/weresync", line 11, in <module>
    sys.exit(main())
  File "/usr/lib64/python3.4/site-packages/weresync/interface.py", line 217, in main
    mount_points, args.rsync_args)
  File "/usr/lib64/python3.4/site-packages/weresync/interface.py", line 129, in copy_drive
    copier.partitions_valid()
  File "/usr/lib64/python3.4/site-packages/weresync/device.py", line 657, in partitions_valid
    target_parts = self.target.get_partitions()
  File "/usr/lib64/python3.4/site-packages/weresync/device.py", line 69, in get_partitions
    raise weresync.exception.DeviceError(self.device, "Non-zero exit code", str(error, "utf-8"))
weresync.exception.DeviceError: ('/dev/sdb', 'Non-zero exit code', 'Error: /dev/sdb: unrecognised disk label\n')
$ sudo parted -s /dev/sdb p
Error: /dev/sdb: unrecognised disk label
Model: Samsung SSD 850 EVO 500G (scsi)
Disk /dev/sdb: 500GB
Sector size (logical/physical): 512B/512B
Partition Table: unknown
Disk Flags: 
```

This fixed the problem: `$ sudo parted /dev/sdb --script -- mklabel gpt` but I would have expected weresync to do this for me.

**Comment by DonyorM** [comment]
Thanks for reporting this!

The target and source drives are represented by the same class in the code, and the source drive needs a disk label in order for certain information to be queried. Target drive shouldn't need that though, I'll see if I can put a fix in.

**Comment by DonyorM** [comment]
Fixed in bb3d078.

Thank you for contributing.
